tests/e2e/util.go

```diff
@@ -805,6 +809,119 @@ func invokeVCenterServiceControl(command, service, host string) error {
 	return nil
 }
 
+//createGC method creates GC and takes WCP host and bearer token as input param
+func createGC(wcpHost string, wcpToken string) {
+
+	transCfg := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
+	}
+
+	client := &http.Client{Transport: transCfg}
+	createGCURL := "https://" + wcpHost + tkgAPI
+	framework.Logf("URL %v", createGCURL)
+	tkg_yaml, err := filepath.Abs(gcManifest + "tkg.yaml")
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	framework.Logf("Takinig yaml from %v", tkg_yaml)
+	gcBytes, err := ioutil.ReadFile(tkg_yaml)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	req, err := http.NewRequest("POST", createGCURL, bytes.NewBuffer(gcBytes))
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	req.Header.Add("Authorization", "Bearer "+wcpToken)
+	req.Header.Add("Accept", "application/yaml")
+	req.Header.Add("Content-Type", "application/yaml")
+
+	resp, err := client.Do(req)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	defer resp.Body.Close()
+
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
+	framework.Logf("API Response status %d", resp.StatusCode)
+	gomega.Expect(resp.StatusCode).Should(gomega.BeNumerically("==", 201))
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	response := string(bodyBytes)
+	framework.Logf(response)
+
+}
+
+//getGC polls for the GC status, returns error if its not in running phase
+func getGC(wcpHost string, wcpToken string, gcName string) error {
+	transCfg := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
+	}
+
+	client := &http.Client{Transport: transCfg}
+	getGCURL := "https://" + wcpHost + tkgAPI + gcName
+	framework.Logf("URL %v", getGCURL)
+
+	req, err := http.NewRequest("GET", getGCURL, nil)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	req.Header.Add("Authorization", "Bearer "+wcpToken)
+
+	waitErr := wait.Poll(pollTimeout, pollTimeout*6, func() (bool, error) {
```

Comment:
polling rate is very huge, may be poll every minute

Reply:
done